cli/wallet/nep5.go

```diff
@@ -66,6 +66,20 @@ func newNEP5Commands() []cli.Command {
 				},
 			},
 		},
+		{
+			Name:      "remove",
+			Usage:     "remove NEP5 token from the wallet",
+			UsageText: "remove --path <path> --token <hash-or-name>",
```

Comment:
Again, do we really a flag? Why not just `remove <token>`?

Reply:
Done!